walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -11,13 +11,64 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 package com.google.sps;
 
-import java.util.Collection;
-
-public final class FindMeetingQuery {
+import java.util.*;
+ 
+public final class FindMeetingQuery { 
+  private ArrayList<TimeRange> viableTimeSlots = new ArrayList<TimeRange>(); 
+ 
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    ArrayList<TimeRange> busyTimes = new ArrayList<TimeRange>();
+    ArrayList<TimeRange> allBusyTimes = new ArrayList<TimeRange>();
+    for(Event e : events) {
+    if(eventContains(e.getAttendees(),request.getAttendees())) { // Ensure event includes requireds
```

Comment:
Can you figure out what the styling issues are here? Can you fix them?

Reply:
And please fix the rest of the file for the issues you have found in line 24 and 25.